src/Halogen/HTML/Properties.purs

```diff
@@ -3,7 +3,7 @@
 -- | standard unrefined versions.
 module Halogen.HTML.Properties
   ( IndexedProp(..)
-  , IProp
+  , IndexedProp
```

Comment:
Maybe we should keep using the `IProp` name too, and remove `IndexedProp` entirely again - it will break just a few less things when we update to this version of Halogen :smile:

It'll also fix the tests.